pkg/grpc/default_api/api_subscribe_test.go

```
@@ -0,0 +1,134 @@
+			stream:           stream,
+			publishResponses: make(map[string]chan *runtimev1pb.SubscribeTopicEventsRequestProcessed),
+		}
+
+		err = apiForTest.SubscribeTopicEvents(stream)
+		assert.Nil(t, err)
+
+		apiForTest.json = jsoniter.ConfigFastest
+
+		go func() {
+			time.Sleep(1 * time.Second)
+			ch, _ := apiForTest.streamer.subscribers["___test||layotto"].publishResponses["1"]
```

golangci-lint

S1005: unnecessary assignment to the blank identifier

(gosimple)